app/grandchallenge/reader_studies/templates/reader_studies/readerstudy_detail.html

```diff
@@ -308,6 +317,36 @@ <h2>Questions</h2>
                         </li>
                     {% endfor %}
                 </ul>
+                <script>
```

Comment:
Do we have a place for local JavaScript? Same pattern as jqfileuploader maybe?

Reply:
Add it as a file in the `js` subdirectory of the reader study templates, then add it to the `script` template block.